src/common/modal/index.jsx

```diff
@@ -12,7 +12,7 @@ const Modal =({ title, show, onClose, onSubmit, children, cname })=> {
     }
     window.addEventListener('keydown', close)
     return () => window.removeEventListener('keydown', close)
-},[])
+},[onClose])
```

Comment:
Just to avoid warning?

Reply:
@Angryman18 can comment.

Reply:
yes this is just to remove dependency warning